pkg/tsdb/postgres/postgres_test.go

```diff
@@ -222,7 +222,7 @@ func TestPostgres(t *testing.T) {
 
 		sql = `
 			INSERT INTO postgres_types VALUES(
-				1,2,3,
+				null,2,3,
```

Comment:
maybe we can have both, nullable and no nullable both tested. insert one more value instead of replacing the current one with null.

Reply:
I added a new column